feat(browser): Document `unregisterOriginalCallbacks` option in `browserApiErrors` integration page (#13859)
getsentry/sentry-docs: docs/platforms/javascript/common/troubleshooting/index.mdx

</Expandable>
</PlatformCategorySection>

<PlatformCategorySection supported={['browser']}>

<Expandable permalink title="SDK causes double invocations of `addEventListener` (e.g. duplicated click events)">

> In very rare cases, the SDK can cause callbacks added via `addEventListener` to an event target (e.g. a button) to be invoked twice.

Reviewer: ```suggestion
In very rare cases, the SDK can cause callbacks added via `addEventListener` to an event target (such as a button) to be invoked twice.
```
